docs/home/howto/DATABASES.md

```diff
@@ -11,7 +11,7 @@ Then configure database server using psql according to your needs and
 
 ## PostgreSQL passwords encryption
 
-Epiphany sets up MD5 password encryption by default. Although PostgreSQL since version 10 is able to use SCRAM SHA-256 password encryption, Epiphany does support this encryption only for one-node Postgresql configuration. HA components like pgbouncer and pgpool are not able to refresh users/passwords list while this encryption is enabled. So those components can be used only with MD5 passwords.
+Epiphany sets up MD5 password encryption by default. Although PostgreSQL since version 10 is able to use SCRAM SHA-256 password encryption, Epiphany does support this encryption only for Postgresql configuration without HA. HA components like pgbouncer and pgpool are not able to refresh users/passwords list while this encryption is enabled. So those components can be used only with MD5 passwords.
```

Comment:
```suggestion
Epiphany sets up MD5 password encryption by default. Although PostgreSQL since version 10 is able to use SCRAM-SHA-256 password encryption, Epiphany does support this encryption only for Postgresql configuration without Pgpool. Pgpool is not able to parse SCRAM-SHA-256 hashes list while this encryption is enabled. Due to limited Pgpool authentication options, it is not possible to maintain the [pool_passwd file](https://www.pgpool.net/docs/42/en/html/auth-methods.html#AUTH-SCRAM) securely, hence it is also not possible to establish a secure connection between components.

Postgresql + Pgbouncer with SCRAM-SHA-256 deployment is possible, but it will require additional manual steps to perform in Pgboucer configuration.
```